routing/update_linux.go

```diff
@@ -17,7 +19,13 @@ func update() ([]Route, error) {
 		if err != nil {
 			return nil, err
 		}
-
+		if len(regexp.MustCompile(`(default)`).FindStringSubmatch(output))!=2 {
```

Comment:
every time this runs, the package recompiles the regular expression needlesly, please precompile outside and reuse the object, like for the `parser` regular expression.